samples/nrf9160/serial_lte_modem/prj.conf

```diff
@@ -42,8 +42,12 @@ CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=8192
 # AT_CMD
 #CONFIG_AT_CMD_LOG_LEVEL_DBG=y
 
+# Device power management
+#CONFIG_DEVICE_POWER_MANAGEMENT=y
```

Comment:
If we want to keep the commented-out configs, we should have a textual comment accompanying them.
Alternatively, we could have a couple of Kconfig options belonging to the sample that `select` these, so that this configuration can be done via Kconfig automatically.

Reply:
The reason to add those commented options is due to questions from customers. I found it's easier to ask them to uncomment/comment out some items, instead of adding and deleting them. :) I'll add textual comments.

Reply:
I see, that makes sense and a textual comment is sure helpful then.